tests/ignite/distributed/test_auto.py

```diff
@@ -69,8 +80,10 @@ def _test_auto_model_optimizer(ws, device):
 
 def test_auto_methods_no_dist():
 
-    _test_auto_dataloader(1, 1)
-    _test_auto_dataloader(1, 1, sampler_name="WeightedRandomSampler")
+    _test_auto_dataloader(1, 1, batch_size=1)
+    _test_auto_dataloader(1, 1, batch_size=10)
```

Comment:
Please, add also the case when num_workers is 1 (same as batch_size argument). You can merge two cases in one:
```python
_test_auto_dataloader(1, 1, batch_size=1, num_workers=1)
```

Reply:
Actually there is no argument `num_workers` for `_test_auto_dataloader()`. Should I improve the function?